public/ballers_new.sql

```diff
@@ -42,8 +42,11 @@ CREATE TABLE `area` (
 
 INSERT INTO `area` (`area_id`, `area_name`, `state_id`, `longitude`, `latitude`) VALUES
 (1, 'Lekki Phase 1', 1, '3.5852', '6.4698'),
-(2, 'Orchid Hotel Road', 1, '3.5433919', '6.4432707'),
-(3, 'Wuse II', 2, '', '');
+(2, 'Orchid Road Hotel', 1, '3.5433919', '6.4432707'),
+(3, 'Ikate', 1, '0', '0'),
+(4, 'Ogombo', 1, '0', '0'),
+(5, 'IIsan', 1, '0', '0'),
```

Comment:
I think both fields (with id 5 and 6) refer to the same location `Ilasan`. Due to the font, it might be difficult to get it right, The first two letters are letter `i` and letter `l`.  You can verify it on the map here: https://www.google.com/maps?q=ilasan&um=1&ie=UTF-8&sa=X&ved=2ahUKEwjrtvGer4fqAhUr5uAKHbiSAkkQ_AUoAXoECBcQAw

Reply:
Guess it was a typo from the person that compiled the excel file. Would fix it.